client/src/django.ts

```diff
@@ -56,7 +57,7 @@ const djangoClient = new Vue({
       const { data } = await apiClient.get(`/sessions/${sessionId}/settings`);
       return data;
     },
-    async setSettings(sessionId: string, settings) {
+    async setSettings(sessionId: string, settings: Object) {
```

Comment:
Something more descriptive would be good. I think it's just `{importpath: string, exportpath: string}`. Could you add that to `types.ts`?